snippets/models.py

```diff
@@ -49,6 +49,46 @@ def __str__(self):
 
 register_snippet(Subject)
 
+class k12Subject(TranslatableMixin, models.Model):
+    name = models.CharField(max_length=255)
+    intro_text = RichTextField(blank=True, null=True)
+    seo_title = models.CharField(max_length=255, null=True, blank=True)
+    search_description = models.CharField(max_length=255, null=True, blank=True)
+    image = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
+    subject_category = models.CharField(max_length=255, choices=K12_CATEGORIES, default='none',
```

Comment:
if you add the boolean on the subject (`is_k12=True`, for eg.), you can just add this to the page using a query, similar to how we do it on the subjects page. feel free to ask for clarification on this if needed